ci-operator/config/openshift/origin/openshift-origin-master.yaml

```diff
@@ -239,6 +239,12 @@ tests:
   commands: TEST_SUITE=openshift/conformance/serial run-tests
   openshift_installer:
     cluster_profile: aws
+- as: e2e-aws-csi
+  commands: TEST_INSTALL_CSI_DRIVERS=aws-ebs TEST_SUITE=openshift/csi run-tests
+  optional: true
+  skip_report: true
```

Comment:
Sorry, I should have been clearer: these should be in the jobs (under `ci-operator/jobs`), not here. `prowgen` will not overwrite them when you regenerate.

Reply:
Will fix, thanks!

Reply:
Fixed, PTAL.

I left the `optional` and `skip_report` options in a separate commit (https://github.com/openshift/release/pull/7131/commits/24afd2914adedb30f4fae12264091494b18a3bc9) to make the review easier.